compiler/rustc_hir/src/hir.rs

```diff
@@ -776,7 +776,6 @@ impl<T> MaybeOwner<T> {
 #[derive(Debug)]
 pub struct Crate<'hir> {
     pub owners: IndexVec<LocalDefId, MaybeOwner<&'hir OwnerInfo<'hir>>>,
-    pub hir_hash: Fingerprint,
```

Comment:
Why this change? This may force re-executions that are unrelated to `crate_hash`.

Reply:
Yes, that's true, making the `hir_crate` query `no_hash` is an unrelated change. Although from the performance results it looks like it does not make much of difference?